conda_forge_feedstock_ops/container_utils.py

```diff
@@ -6,50 +6,12 @@
 from collections.abc import Iterable
 from typing import Callable, Optional
 
-from ._version import __version__
+from conda_forge_feedstock_ops.settings import FeedstockOpsSettings
 
 logger = logging.getLogger(__name__)
 
 DEFAULT_CONTAINER_TMPFS_SIZE_MB = 6000
 
-CONTAINER_PROXY_MODE = os.environ.get(
-    "CF_FEEDSTOCK_OPS_CONTAINER_PROXY_MODE", "false"
-).lower() in ("yes", "true", "t", "1")
-"""
-Whether to use a proxy that is locally configured for all requests inside the container.
-Set the environment variable `CF_FEEDSTOCK_OPS_CONTAINER_PROXY_MODE` to 'true' to enable this feature.
-"""
-
-PROXY_IN_CONTAINER = os.environ.get(
-    "CF_FEEDSTOCK_OPS_PROXY_IN_CONTAINER", "http://host.docker.internal:8080"
-)
-"""
-The hostname of the proxy to use in the container.
-The default value of 'http://host.docker.internal:8080' is the default value for Docker Desktop on Windows and macOS.
-It also works for OrbStack.
-
-For podman, use http://host.containers.internal:8080.
-For GitHub Actions, use http://172.17.0.1:8080, see https://stackoverflow.com/a/65505308
-"""
-
-
-def get_default_container_name():
```

Comment:
This function has been removed. That is an API change.

Reply:
although I did not find any usages via the GitHub Code search, I restored this method in 5146cf39f6cc22b408b40613d7cd8b116193ca9d